src/afl-fuzz-globals.c

```diff
@@ -75,7 +76,7 @@ list_t afl_states = {0};
 
 /* Initializes an afl_state_t. */
 
-void afl_state_init(afl_state_t *afl) {
+void afl_state_init(afl_state_t *afl, char **envp) {
```

Comment:
We can think about moving this out of the init but it's fine if we do it later, also

Reply:
I thought of making it a new function. I can do that if you say. :)